solidity/contracts/AttributeCheckpointFraud.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+import {TREE_DEPTH} from "./libs/Merkle.sol";
+import {CheckpointLib, Checkpoint} from "./libs/CheckpointLib.sol";
+import {CheckpointFraudProofs} from "./CheckpointFraudProofs.sol";
+
+enum FraudType {
+    NOT_PROVEN,
+    Whitelist,
```

Comment:
nit: consistent formatting of enum

Reply:
intentionally cased this differently because its not really a valid type but is the default in storage